woocommerce/woocommerce
# Shipping Settings: CSS/HTML tweaks and Modal changes to Shipping Zones

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR does the following:

* Shipping Zone settings HTML/CSS tweaks according to new designs in paJDVv-2Ew-p2.
* Combines the Add Shipping Zone Method modal with the Configure modal such that adding a new method can be done in a single flow.
* Normalizes modal widths so they don't vary as users progress through a flow.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Pull down this branch locally and `pnpm install` or visit the Jurassic Ninja link below
2. Go to WooCommerce > Settings > Shipping
3. Add a few shipping zones and methods
4. See the following visual changes and compare against designs in paJDVv-2Ew-p2:
* Call to action button is primary colors
* Shipping methods are now tags
* Row actions now appear at the end of the row and no longer appear on hover
* Updated call-to-action button placement
* Tooltips replaced by simple help text
* Adding a new shipping zone method in a single flow
* Modal widths are uniform



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update Shipping settings with visual CSS tweaks

#### Comment 

</details>
